stripevpaypal.md

```diff
@@ -0,0 +1,115 @@
+# Stripe and PayPal Payment Setup on Squarespace  
+
+****Authored by:****  [Ryan Lang](https://github.com/langrh), 1st yr. college    
+
+****Last Updated:**** September 21, 2023
+
+## Overview
+
+Stripe and PayPal are two payment types on Squarespace, each with their own pros and cons. However, both can be used simultaneously on your website. This guide will explain how to set up and use both payment processors, along with a comparison between them.
+
+## How to Set Up
+
+1. On Squarespace, navigate to *Commerce -> Payments*.
+2. Once there, PayPal and Stripe will show up as payment methods.
+3. For both, an account must be linked to use.
+4. To create an account for Stripe, navigate to [Stripe Registration](https://dashboard.stripe.com/register).
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.
+
+## Stripe and PayPal Payment Comparison on Squarespace
+
+| Aspect                       | PayPal                                                                                                    | Stripe                                                                                     |
+|------------------------------|-----------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
+| **Transaction Fees**         | Varies by user's location.                                                                                | Varies by country.                                                                         |

```

Comment:
For Paypal, does that mean it also varies state to state/city to city? And the only variation for Stripe is based on country?

Reply:
Updated Transaction Fees.